lib/rspec/rails/view_rendering.rb

```diff
@@ -39,18 +39,50 @@ def render_views?
         self.class.render_views? || !controller.class.respond_to?(:view_paths)
       end
 
-      # Delegates find_all to the submitted path set and then returns templates
-      # with modified source
-      #
-      # @private
-      class EmptyTemplateResolver < ::ActionView::FileSystemResolver
+      class EmptyTemplateResolverFactory
```

Comment:
Agree on both the allocation and naming concerns. How does `EmptyTemplateResolver.build` sound?